src/Events/User/Job/AbstractJobEvent.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace Qless\Events\User\Job;
+
+use Qless\Events\User\AbstractEvent;
+use Qless\Jobs\BaseJob;
+
+abstract class AbstractJobEvent extends AbstractEvent
+{
+    private $job;
+
+    /**
+     * AbstractJobEvent constructor.
+     * @param $source
```

Comment:
PHPDoc tag `@param` has invalid value ($source).

```suggestion
     * @param object $source
```